### PR TITLE
Ignore different iconUrls.

### PR DESCRIPTION
**Pull request by mmulder72**
Added the following paths to ignore iconUrls:
issuelinks.inwardIssue.fields.issuetype.iconUrl
issuelinks.inwardIssue.fields.priority.iconUrl
issuelinks.outwardIssue.fields.issuetype.iconUrl
issuelinks.outwardIssue.fields.priority.iconUrl
issuelinks.outwardIssue.fields.status.iconUrl
subtasks.fields.issuetype.iconUrl
subtasks.fields.priority.iconUrl

I tested on a couple specific JIRA cases to verify that it works.

### CODE REVIEW

File: compare/go-jira.py [hunk 1]
```diff
@@ -86,17 +86,40 @@
...
+                   'comment.comments.id',
+                   'comment.comments.self',
+                   'comment.comments.updateAuthor.emailAddress',
+                   'comment.comments.updated',
+                   'components.id',
+                   'components.self',
+                   'creator',
+                   'fixVersions.id',
+                   'fixVersions.self',
+                   'issuelinks.inwardIssue.fields.issuetype.iconUrl',
```
Comment (jbapple-cloudera) [inline compare/go-jira.py:101]: Could we just put iconUrl in IGNORABLE_KEYS?

Comment (mmulder72) [reply compare/go-jira.py:101]: Not without some other code changes. It looks like IGNORABLE_KEYS is only used when comparing dictionary fields.

Comment (jbapple-cloudera) [reply compare/go-jira.py:101]: I think the path issuelinks.inwardIssue.fields.issuetype.iconUrl is created in ￼ compare_and_print_fields by iterating through dictionaries. I think each iteration uses a key and adds a dot to the path.

Comment (mmulder72) [reply compare/go-jira.py:101]: The IMPALA cases are no longer in issues-test.apache.org, so I can't test that now.

Comment (jbapple-cloudera) [reply compare/go-jira.py:101]: Sorry for that. They are back up now.

**Pull request closed without merging**